packages/js/core-client/src/PolywrapCoreClient.ts

```diff
@@ -316,10 +330,16 @@ export class PolywrapCoreClient implements CoreClient {
         errors: errors.length === 0 ? undefined : errors,
       };
     } catch (error: unknown) {
+      const unknownQueryErrorToWrapError = (e: Error): WrapError =>
+        new WrapError((e as Error)?.message, {
```

Comment:
Should we add a default string here if a message property is not found, instead of `undefined`? Maybe "client.query: no error message found" or something like that?

Reply:
2 points to convey:
- from within query method
- no error message found / unknown error obj caught

Reply:
Nice catch. I changed it to `e?.message ?? "client.query: no error message found"`